### PR TITLE
batch invitations

### PR DESCRIPTION
**Pull request by mcginty**
screenie

### CODE REVIEW

File: res/menu/text_secure_normal.xml [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
 
     <item android:title="@string/arrays__my_identity_key"
-          android:id="@+id/menu_my_identity"
-          android:icon="@android:drawable/ic_menu_view" />
+          android:id="@+id/menu_my_identity" />
 
     <item android:title="@string/text_secure_normal__menu_settings"
-          android:id="@+id/menu_settings"
-          android:icon="@android:drawable/ic_menu_preferences" />
+          android:id="@+id/menu_settings" />
```
Comment (moxie0) [inline res/menu/text_secure_normal.xml]: why are you getting rid of these GB icons?

File: res/layout/reminder_header.xml [hunk 2]
```diff
@@ -34,29 +36,23 @@
...
+                android:textColor="@color/white"
+                android:text="@string/reminder_header_close_button" />
+
+        <Button android:id="@+id/accept"
+                style="@style/Widget.AppCompat.Button.Borderless"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textColor="@color/white"
+                tools:text="ENABLE" />
+    </LinearLayout>
```
Comment (moxie0) [inline res/layout/reminder_header.xml]: I'd like to go back to turning the entire view into the tap target, with an X in the top-right, FB-messenger style.

**Comment by moxie0** [comment]
How did you solve the issue of only displaying non-ts users for sms invite?

File: res/menu/text_secure_normal.xml (hunk 1 above)
Comment (mcginty) [inline res/menu/text_secure_normal.xml]: appcompat doesn't use them any more

**Comment by mcginty** [comment]
I didn't.

**Comment by moxie0** [comment]
that's a pretty essential part of this feature

File: res/layout/reminder_header.xml (hunk 2 above)
Comment (mcginty) [inline res/layout/reminder_header.xml]: I'd like to get better graphic assets done before that happens, otherwise it looks pretty boring imo.

Comment (moxie0) [inline res/layout/reminder_header.xml]: i prefer what we have now over this, feels like a step backwards

Comment (mcginty) [inline res/layout/reminder_header.xml]: subjectively disagree but sure i'll go back to the original one

**Comment by McLoo** [comment]
Line break before the URL ? Looks a bit scatt
ered...

**Comment by mcginty** [comment]
The only way to cleanly do the filtering for TS users is by asynchronously checking whether the number is in the directory or not already and just disabling the checkmarks for those users. I don't know of a way to filter the query directly - the ways I tried didn't seem to have any effect.

@McLoo I like where you're going with this

**Pull request closed without merging**